common/build/eslint-config-fluid/minimal-deprecated.js

```diff
@@ -7,8 +7,8 @@
  * Shared list of permitted imports for configuring and override the `import/no-internal-modules` rule.
  */
 const permittedImports = [
-	// Within Fluid Framework allow import of '/internal'.
-	"@fluidframework/*/internal",
+	// Within Fluid Framework allow import of '/internal' from other FF packages.
+	"@fluid*/*/internal",
 
```

Comment:
I did think we wanted to leave this just to @fluidframework. We don't need (AFAIK), to apply trimming to other scopes. If those cases exist they are rare and can be case-by-case allowed.